imod/tests/test_mf6/test_mf6_chd.py

```diff
@@ -192,3 +193,20 @@ def test_write_concentration_period_data(head_fc, concentration_fc):
             assert (
                 data.count("2") == 1755
             )  # the number 2 is in the concentration data, and in the cell indices.
+
+
+@pytest.mark.usefixtures("imod5_dataset")
+def test_from_imod5(imod5_dataset):
+    target_dis = StructuredDiscretization.from_imod5_data(imod5_dataset)
+
+    chd_layer_3 = imod.mf6.ConstantHead.from_imod5_data(
+        "chd-3",
```

Comment:
Hmmm, I remember now that the projectfile reader inferred separate CHD packages for each layer. I have to investigate again why that was the case, as I'm not really a fan of this.

Reply:
the method "from_imod5_data" in the previous version of this PR is now renamed "from_imod5_ibound" and  only focusses on creating a chd package on the locations where ibound == -1. The starting head is used for the head in this chd package. Another issue (#1087) was created to import the other chd packages.